workflows/offline_ml_diags/tests/test__ml_prediction.py

```diff
@@ -1,219 +0,0 @@
-import xarray as xr
```

Comment:
Probably don't need everything in here but don't we lose test coverage by not porting these over to tests of `predict_function`?

Reply:
Yes, it looks like what predict_function actually does (other than predict, which we already have as `model.predict`) is insert grid data and insert zero-values for (p)Q(1/2). I'll refactor these tests to test predict_function for these behaviors instead of completely removing them.

Reply:
Done.